e2e/browser/test-app/package.json

```diff
@@ -16,10 +16,11 @@
     "start": "next start"
   },
   "dependencies": {
+    "@inrupt/internal-playwright-testids": "file:../../../../typescript-sdk-tools/packages/internal-playwright-testids",
```

Comment:
Is this intentional, or an artifact of local testing?

Reply:
this was actually already like this previous to my PR, my changes just ended up moving it to the top of the dependencies. 

but I will update to the most recent published version of @inrupt/internal-playwright-testids